### PR TITLE
[AD-78] Update custom-namespaces.yaml

### PR DESCRIPTION
**Pull request by sergiosonline**
Adding consolidated_ad_metrics under new Ads superset

### CODE REVIEW

File: custom-namespaces.yaml [hunk 1]
```diff
@@ -506,6 +506,19 @@ revenue_private:
...
+  pretty_name: Ads
+  owners:
+    - cmorales@mozilla.com
+    - sbetancourt@mozilla.com
+  spoke: looker-spoke-private
+  views:
+    consolidated_ad_metrics:
+      type: table_view
+      tables:
+        - table: mozdata.ads.consolidated_ad_metrics
```
Comment (curtismorales) [inline custom-namespaces.yaml:521]: We should also pull in the Pacific Time table and the hourly table

```suggestion
        - table: mozdata.ads.consolidated_ad_metrics
        - table: mozdata.ads.consolidated_ad_metrics_daily_pt
        - table: mozdata.ads.consolidated_ad_metrics_hourly
```

Comment (sergiosonline) [reply custom-namespaces.yaml:521]: @curtismorales : Makes sense. Do we need to add any conditions or filters to separate these three out? Looking at https://github.com/mozilla/lookml-generator/blob/5c771f8c49f39788a597f919f678345904f6abd2/custom-namespaces.yaml#L229

Comment (sergiosonline) [reply custom-namespaces.yaml:521]: Yeah, let's do this for now. We can optimize later. Thank you.

**Review by curtismorales — APPROVED** [review]
Looks good!

**Pull request merged**